src/components/molecules/TableHeader/components/EditTableTitleModal/EditTableTitleModal.tsx

```diff
@@ -0,0 +1,170 @@
+  }, [formDefaultValues, reset, title]);
+
+  // use useAsyncFn for easier error handling, instead of state hook
+  const [{ error }, updateTables] = useAsyncFn(
+    async (values: EditTableForm) => {
+      if (!user || !tableOfUser || !venueId) return;
+
+      const updatedTable = {
+        ...tableOfUser,
+        title: values.title,
+        subtitle: values.subtitle,
+        capacity: values.capacity,
+      };
+
+      return await firebase.functions().httpsCallable("venue-updateTables")({
+        venueId,
+        tables,
+        updatedTable,
+      });
+    },
+    [tableOfUser, tables, user, venueId]
+  );
+
+  const values = watch();
+
+  const onSubmit = useCallback(async () => {
+    await updateTables(values).then(() => {
+      onHide();
+    });
+  }, [onHide, updateTables, values]);
+
+  return (
+    <Modal show={isShown} onHide={onHide}>
+      <Modal.Body>
+        <form onSubmit={handleSubmit(onSubmit)} className="EditTableTitleModal">
+          <InputField
+            ref={register({ required: true })}
+            name="title"
+            defaultValue={title}
+            containerClassName="EditTableTitleModal__input--spacing"
+            placeholder="Table topic"
+          />
+          {errors.title && errors.title.type === "required" && (
+            <span className="input-error">Topic is required</span>
+          )}
+
+          <InputField
+            ref={register}
+            name="subtitle"
+            containerClassName="EditTableTitleModal__input--spacing"
+            placeholder="Describe this table (optional)"
+          />
+
+          <div className="EditTableTitleModal__capacity">
+            <label className="EditTableTitleModal__max-capacity">
+              Number of seats (max {MAX_TABLE_CAPACITY})
+            </label>
+            <InputField
+              ref={register({
+                required: true,
+                max: MAX_TABLE_CAPACITY,
+                min: MIN_TABLE_CAPACITY,
+              })}
+              name="capacity"
+              type="number"
+              min={MIN_TABLE_CAPACITY}
+              max={MAX_TABLE_CAPACITY}
+              placeholder="Max seats"
+            />
+          </div>
+          {errors.capacity?.type === "required" && (
+            <span className="input-error">Capacity is required</span>
+          )}
+          {(errors.capacity?.type === "min" ||
+            errors.capacity?.type === "max") && (
+            <span className="input-error">
+              Capacity must be between
+              {` ${MIN_TABLE_CAPACITY} and ${MAX_TABLE_CAPACITY}`}
+            </span>
+          )}
+          {error && <label className="input-error">{error.message}</label>}
+
+          <div className="EditTableTitleModal__footer-buttons">
+            <button
+              className="btn btn-centered btn-secondary"
+              onClick={onHide}
+              type="reset"
+            >
+              Cancel
+            </button>
```

Comment:
Should we be using `type="reset"` on this button? Or should it just be `type="button"`?

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type
  - > `reset`: The button resets all the controls to their initial values, like `<input type="reset">`. (This behavior tends to annoy users.)
  - > `button`: The button has no default behavior, and does nothing when pressed by default. It can have client-side scripts listen to the element's events, which are triggered when the events occur.

Personally I feel like it should just be `type="button"`. Also, prefer to have the `type` prop on `button`s at the start:

```suggestion
            <button
              type="button"
              className="btn btn-centered btn-secondary"
              onClick={onHide}
            >
              Cancel
            </button>
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/eabf5f4a15571cf2e4e13c2c78ce8659071752c3